instructor/common.py

```diff
@@ -1670,6 +1697,24 @@ def facs_histograms_edit(request, assignment, sample_prep, sp):
     )
 
 
+@assignment_selected
+@check_assignment_owner
+@login_required
+def select_images(request):
+    mapping_id = request.POST.get('mapping_pk')
+    image_pk_list = request.POST.getlist('image_pk_list[]')
+    instance = get_object_or_404(models.MicroscopyImageMapping, pk=mapping_id)
+    for image_pk in image_pk_list:
+        selected_image = get_object_or_404(models.MicroscopyImage, pk=image_pk)
+
+        instance.images.add(selected_image)
+    instance.save()
+
+    return HttpResponse('complete')
```

Comment:
We should not return anything for content here unless it has meaning to a client. Here it seems like the default `200` status code is enough so we should return `HttpResponse()` instead